cards/spells/shocking-grasp.tex

```diff
@@ -6,20 +6,19 @@
 
 \Flavor{Shroud your hands in a crackling field of lightning.}
  
-\Action[Attack, Somatic, Verbal]{CRB}{369}{2}{Cast}
-
 Target 1 creature you can touch.
 
-If the target is wearing metal armor or is made of metal.
+\Action[Attack, Somatic, Verbal]{CRB}{369}{2}{Cast}
 
-% TODO: CheckRoll needs a way to add further modifiers (like +1)
-\CheckRoll{Spell Attack}{spell atk} \(+ 1\)
+If target is wearing metal armor or made of metal.
+
+\CheckRoll{Spell Attack}{int, prof, 1}
 % TODO: how to express persistent damage?
 \DamageRoll{Electricity}{2d12}{} \DamageRoll{Electricity}{1d4}{} (persistent)
 
 \pfsymbol{critical-success} Double the initial damage, but not the persistent damage.
 
-Otherwise
+\Action[Attack, Somatic, Verbal]{CRB}{369}{2}{Cast}
 
-\CheckRoll{Spell Attack}{spell atk}
+\CheckRoll{Spell Attack}{int, prof}
```

Comment:
Hm, why is this always INT? I think a Wizard uses INT, but a Druid would use WIS, wouldn't they?

I chose to refer to Spell Attack here because that's also a field on the character sheet...

Reply:
Yes the attribute used for the spell attack roll depends on the tradition of the spell. Some spells can come from multiple traditions, then it matters how the character got the spell. Characters can also get spells from different traditions, then they use different spell attack rolls. See https://2e.aonprd.com/Rules.aspx?ID=325

Reply:
Regarding what to write on the card, if I remember our discussion from yesterday correctly, we kind of agreed:

- for the "cast" action it makes sense to just put "spell dc" or "spell attack roll" and let the player know what that is for their character. probably they don't even need to copy that number over to the card.
- for the "strike" action it makes sense to figure out the number per weapon and write it down somewhere, presumably on the card, because the card plays the role of the "weapons" entry of a character sheet?

I originally thought these two should be treated the same, but then you convinced me that weapons are actually so much more complicated that it is more important to precompute and write down the bonus there. In particular, weapons are more complicated because proficiency, item bonus, and melee vs. ranged.